_posts/2020_icml/2020-05-24-Braga.md

```diff
@@ -0,0 +1,17 @@
+---
+paperId: 35
+author: Heitor Rapela Medeiros, Pedro H. M. Braga, Hansenclever F. Bassani
+publicationauthor: Braga, P. H. M. et al
+title: Deep Clustering Self-Organizing Maps with Relevance Learning
+pdf: --
+poster: Poster_Pedro_Braga
```

Comment:
@anthxnyR the link to this poster is not resolving correctly, its showing a 404 page not found.